server/common/CommandLine.java

```diff
@@ -74,9 +74,13 @@ public CommandLine command(Command.Parser<?> parser) {
             String arg = args[i];
             if (arg.startsWith(Option.PREFIX)) {
                 int index = arg.indexOf("=");
-                if (index != -1) options.add(new Option(arg.substring(2, index), arg.substring(index + 1)));
-                else if (args[i + 1].startsWith(Option.PREFIX)) options.add(new Option(arg.substring(2)));
-                else options.add(new Option(arg.substring(2), args[++i]));
+                if (index != -1) {
+                    options.add(new Option(arg.substring(2, index), arg.substring(index + 1)));
+                } else if (i + 1 == args.length || args[i + 1].startsWith(Option.PREFIX)) {
```

Comment:
fix a bug where we can get an array out of bounds exception if  `i + 1` does not exist in the array